src/components/common-components/dropdown-buttons/dropdown-buttons.component.tsx

```diff
@@ -72,7 +72,10 @@ export function DropdownButtons({
               <div
                 className="dropdown-button"
                 key={item.label}
-                onClick={item.action}
+                onClick={(): void => {
```

Comment:
Zrobiłem tak, żeby przy każdym kliknięciu na opcję z dropdown'u on się zamykał.

Reply:
Może trzymajmy sie tych useCallbacków?

Reply:
Poprawię to

Reply:
Niestety nie da się, bo item jest wyciągany za pomocą `map`.